cabal.project

```diff
@@ -28,8 +28,9 @@ package cardano-cli
 package cardano-git-rev
   ghc-options: -Werror
 
-package cardano-node
-  ghc-options: -Werror
+-- FIXME: this needs to be enabled again!
```

Comment:
I would mention that this is because there are some incomplete patterns related to `Orphans/Network`